src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/SystemNotifyApplicantDisputeFormOverdueTask.java

```diff
@@ -0,0 +1,99 @@
+import static uk.gov.hmcts.divorce.systemupdate.service.CcdSearchService.AOS_RESPONSE;
+import static uk.gov.hmcts.divorce.systemupdate.service.CcdSearchService.DATA;
+import static uk.gov.hmcts.divorce.systemupdate.service.CcdSearchService.ISSUE_DATE;
+import static uk.gov.hmcts.divorce.systemupdate.service.CcdSearchService.STATE;
+
+@Component
+@Slf4j
+public class SystemNotifyApplicantDisputeFormOverdueTask implements Runnable {
+
+    static final String NOTIFICATION_SENT_FLAG = "disputeNotSubmittedNotificationSent";
+    private static final String SUBMIT_EVENT_ERROR = "Submit event failed for case id: {}, continuing to next case";
+    private static final String DESERIALIZATION_ERROR = "Deserialization failed for case id: {}, continuing to next case";
+    private static final String CCD_SEARCH_ERROR = "NotifyApplicantDisputeFormOverdue schedule task stopped after search error";
+    private static final String TASK_CONFLICT_ERROR =
+        "NotifyApplicantDisputeFormOverdue scheduled task stopping due to conflict with another running task";
+
+    @Autowired
+    private CcdSearchService ccdSearchService;
+
+    @Autowired
+    private CcdUpdateService ccdUpdateService;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private IdamService idamService;
+
+    @Autowired
+    private AuthTokenGenerator authTokenGenerator;
+
+    @Override
+    public void run() {
+        log.info("NotifyApplicantDisputeFormOverdue scheduled task started");
+
+        final User user = idamService.retrieveSystemUpdateUserDetails();
+        final String serviceAuth = authTokenGenerator.generate();
+
+        try {
+            final BoolQueryBuilder query =
+                boolQuery()
+                    .must(matchQuery(STATE, Holding))
+                    .must(matchQuery(AOS_RESPONSE, DISPUTE_DIVORCE))
+                    .filter(rangeQuery(ISSUE_DATE).lte(LocalDate.now().minusDays(37)))
+                    .mustNot(matchQuery(String.format(DATA, NOTIFICATION_SENT_FLAG), YesOrNo.YES));
+
+            ccdSearchService.searchForAllCasesWithQuery(Holding, query, user, serviceAuth)
+                .forEach(caseDetails -> notifyApplicant(caseDetails, user, serviceAuth));
+
+            log.info("NotifyApplicantDisputeFormOverdue scheduled task complete.");
+        } catch (final CcdSearchCaseException e) {
+            log.error(CCD_SEARCH_ERROR, e);
+        } catch (CcdConflictException e) {
+            log.error(TASK_CONFLICT_ERROR, e);
+        }
+    }
+
+    private void notifyApplicant(CaseDetails caseDetails, User user, String serviceAuth) {
+        try {
+            final CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
+            if (!caseData.getApplication().hasApplicantBeenNotifiedDisputeFormOverdue()
+                && !caseData.getApplication().getIssueDate().plusDays(37).isAfter(LocalDate.now())) {
```

Comment:
why do you need these checks if they are in the ES query?

Reply:
tbh, I added the date check later as an extra layer of validation. not sure that it's needed really

Reply:
if they're not needed them get rid